src/client/client.go

```diff
@@ -153,8 +154,84 @@ func (c APIClient) SetMaxConcurrentStreams(n int) {
 	c.streamSemaphore = make(chan struct{}, n)
 }
 
+// PachTransientDialOptions is a helper returning a slice of grpc.Dial options
+// such that
+// - TLS is disabled
+// - Dial is synchronous: the call doesn't return until the connection has been
+//                        established and it's safe to send RPCs
+//
+// This is primarily useful for Pachd and Worker clients
+func PachTransientDialOptions() []grpc.DialOption {
+	return []grpc.DialOption{
+		grpc.WithInsecure(),
+
+		// Don't return from Dial() until the connection has been established
+		grpc.WithBlock(),
+
+		// If no connection is established in 10s, fail the call
+		grpc.WithTimeout(10 * time.Second),
+	}
+}
+
+// PachDurableDialOptions is a helper returning a slice of grpc.Dial
+// options such that
+// - TLS is disabled
+// - Dial is synchronous: the call doesn't return until the connection has been
+//                        established and it's safe to send RPCs
+// - The connection is durable: the connection is maintained even when no RPCs
+//                              are sent. This may avoid timeout where the
+//                              client has to re-connect after a period of
+// 															idleness.
+//
+// This is primarily useful for Pachd and Worker clients
+func PachDurableDialOptions() []grpc.DialOption {
+	return append(PachTransientDialOptions(),
+		// Send keepalives to the server, even if no RPCs are pending
+		grpc.WithKeepaliveParams(keepalive.ClientParameters{
+			// If no activity is seen in 10s, ping server
+			Time: 10 * time.Second,
```

Comment:
Not really. I think in most places with timeouts we were waiting 5s, but that seemed a little short to me. But I'd be happy to use that.